docker/bin/start-api

```diff
@@ -32,4 +33,8 @@ if [[ -n "${GUNICORN_LOGGER_CLASS}" ]]; then
     GUNICORN_OPTIONS+=(--logger-class "${GUNICORN_LOGGER_CLASS}")
 fi
 
+if [[ "${RELOAD_ENABLED}" == 'yes' ]]; then
+    GUNICORN_OPTIONS+=(--reload)
+fi
```

Comment:
adding --reload to gunicorn

Reply:
Currently we are starting the API with the `reloader` script https://github.com/ansible/galaxy_ng/blob/62871b98e50499ef369f9165e5e4f9cbc0051dd3/dev/compose/aap.yaml#L202 

Will this replace the current reloader? or both can work together?

Reply:
I haven't tested yet, the idea of this PR is having a way to reload from other dev environments: operator, aap-dev, ...

Reply:
I tested it locally,
it can replace and it can work together.
So far it is disabled on this PR, should I enable it by adding `RELOAD_ENABLED: yes` on the compose files?